golang/trace/zipkin_tracereporter.go

```diff
@@ -64,18 +64,20 @@ func (r *ZipkinTraceReporter) Report(
 }
 
 // buildZipkinSpan builds zipkin span based on tchannel span.
-func buildZipkinSpan(span tc.Span, annotations []tc.Annotation, binaryAnnotations []tc.BinaryAnnotation, name string, endpoint *tc.Endpoint) *tcollector.Span {
+func buildZipkinSpan(span tc.Span, annotations []tc.Annotation, binaryAnnotations []tc.BinaryAnnotation, targetEndpoint tc.TargetEndpoint) *tcollector.Span {
```

Comment:
I like to use `NewTypeName` when `TypeName` is defined by us and it is the constructor. In this case, it's not *the* constructor since it's a Thrift type, so I don't mind the build name here.